papers/ian_kenney/ian_kenney.rst

```diff
@@ -109,21 +109,22 @@ Software has become increasingly essential to research. In many areas, it underl
-With the publication of “The FAIR Guiding Principles for scientific data management and stewardship” in 2016 and the follow-up FAIR Principles for Research Software in 2022, it has become increasingly acknowledged that abiding by the principles of FAIR is crucial to promoting robust, reproducible, and efficient scientific discovery and innovation  :cite:`chue_hong_fair_2021,wilkinson_fair_2016`. We believe that extending FAIR principles to include open-source software not only significantly advances that goal, but furthermore is necessary for transparent research. Open sharing of code brings a number of substantial benefits to the scientific community. For example, scientists can accurately replicate a given methodology or re-use previous code, reducing duplication of effort and reducing the risk of implementation errors. Indeed, the molecular simulation community in particular has made a concerted effort over recent years to encourage the open sharing of scientific codes :cite:`walters_code_2020`. For example, as of July 2022, over 4700 GitHub repositories containing Python code that makes use of MDAnalysis :cite:`michaud-agrawal_mdanalysis_2011, gowers_mdanalysis_2016` have been made publicly available.
+Despite the importance of software, it is typically not central to the publication peer review process in many scientific fields. Consequently, scientific code frequently fails to meet the basic tenets of FAIR: findability, accessibility, interoperability, and reusability :cite:`chue_hong_fair_2022,wilkinson_fair_2016`. 
+With the publication of “The FAIR Guiding Principles for scientific data management and stewardship” in 2016 and the follow-up "FAIR Principles for Research Software (FAIR4RS Principles)" in 2022, it has become increasingly acknowledged that abiding by the principles of FAIR is crucial to promoting robust, reproducible, and efficient scientific discovery and innovation  :cite:`chue_hong_fair_2022,wilkinson_fair_2016`. We believe that extending FAIR principles to include open-source software not only significantly advances that goal, but furthermore is necessary for transparent research. Open sharing of code brings a number of substantial benefits to the scientific community. For example, scientists can accurately replicate a given methodology or re-use previous code, reducing duplication of effort and reducing the risk of implementation errors. Indeed, the molecular simulation community in particular has made a concerted effort over recent years to encourage the open sharing of scientific codes :cite:`walters_code_2020`. For example, as of July 2022, over 4700 GitHub repositories containing Python code that makes use of MDAnalysis :cite:`michaud-agrawal_mdanalysis_2011, gowers_mdanalysis_2016` have been made publicly available [#]_.
```

Comment:
I also added some quotes around the second article title because that was bothering me; unless there was a reason only the first title was quoted?